src/gafaelfawr/verify.py

```diff
@@ -87,13 +87,17 @@ def verify_internal_token(self, token: OIDCToken) -> OIDCVerifiedToken:
             raise InvalidTokenError(str(e))
         return self._build_token(token.encoded, payload)
 
-    async def verify_oidc_token(self, token: OIDCToken) -> OIDCVerifiedToken:
+    async def verify_oidc_token(
+        self, token: OIDCToken, verify_uid=True
+    ) -> OIDCVerifiedToken:
         """Verifies the provided JWT from an OpenID Connect provider.
 
         Parameters
         ----------
         token : `gafaelfawr.models.oidc.OIDCToken`
             JWT to verify.
+        verify_uid : boolean
```

Comment:
```suggestion
        verify_uid : `bool`
```

Reply:
fixed in 4630a787a42b7e507a9eb7948cad8853bf997d3e